tests/test_subdirectory.py

```diff
@@ -209,13 +220,14 @@ def test_new_version_uses_subdirectory(tmp_path_factory):
         git("tag", "v2")
 
     # Finally, update the generated project
-    copier.copy(dst_path=project_path, defaults=True, overwrite=True)
+    copier.copy(dst_path=project_path, defaults=True, overwrite=True, conflict=conflict)
     assert "_commit: v2" in (project_path / ".copier-answers.yml").read_text()
 
-    # Assert that the README still exists, and was force updated to "upstream version 2"
+    # Assert that the README still exists, and the conflicts were handled
+    # correctly.
     assert (project_path / "README.md").exists()
     with (project_path / "README.md").open() as fd:
-        assert fd.read() == "upstream version 2\n"
+        assert fd.read() == readme
```

Comment:
According to what I see in this test, you should assert that a `README.md.rej` file exists when in `rej` mode. This way, we make sure that both modes detect the same conflict, and just its output mode changes.

Reply:
Done!